CHANGELOG.md

```diff
@@ -94,6 +94,13 @@ Note that 3.1.0 will be the first API stable release and interfaces in this rele
   * The configuration element `seqan3::search_cfg::mode` does not exist anymore.
     You can replace it by directly using one of the above mentioned "hit strategy" configuration elements
     ([\#1639](https://github.com/seqan/seqan3/pull/1639)).
+  * The configuration element `seqan3::search_cfg::output` does not exist anymore. It has been replaced by the
+    individual configuration elements
+    * `seqan3::search_cfg::output_query_id`
+    * `seqan3::search_cfg::output_reference_id`
+    * `seqan3::search_cfg::output_reference_begin_pos`
```

Comment:
Did we decided to call this pos? What is wrong with position?

Reply:
I made them correspond to the member functions of the search result. There it's called pos but it might have been that we decided for position. I would change the `pos` -> `position` in a follow up PR for both